app/javascript/iiif-timeliner-styles.css

```diff
@@ -1,3 +1,19 @@
+/* 
```

Comment:
@joncameron @mcwhitaker What should the license say about this and the timeliner pack since it was work for hire?

Reply:
Good question... I think @mcwhitaker can speak to that better than I could.

Reply:
I went ahead and merged this PR and we can make a smaller PR to fix these two files if we need to.